services/api/src/utils/tokens.js

```diff
@@ -13,35 +13,39 @@ const secrets = {
   user: config.get('JWT_SECRET'),
 };
 
-function createTemporaryToken(claims) {
+function createTemporaryToken(payload) {
   return jwt.sign(
     {
-      ...claims,
+      ...payload,
       kid: 'user',
       exp: Math.floor(Date.now() / 1000) + expiresIn.temporary,
     },
     secrets.user
   );
 }
 
-function createAuthToken(fields) {
-  const jti = generateTokenId();
+function createAuthToken(payload) {
+  return jwt.sign(payload, secrets.user);
+}
 
-  const payload = {
+function createAuthTokenPayload(fields) {
+  return {
+    jti: generateTokenId(),
     iat: Math.floor(Date.now() / 1000),
-    jti,
     type: 'user',
     kid: 'user',
     exp: Math.floor(Date.now() / 1000) + expiresIn.regular,
     ...fields,
   };
+}
 
-  const token = jwt.sign(payload, secrets.user);
-
-  return {
-    token,
-    payload,
-  };
+function createTestToken(user) {
+  const payload = createAuthTokenPayload({
+    sub: user.id,
+    ip: '127.0.0.1',
+    userAgent: 'testing library',
+  });
+  return createAuthToken(payload);
 }
```

Comment:
should this be here ? feel like real code and testing utils should be separated

Reply:
ya ok good point, just moved that code